fr/01_0_Presentation_du_Bitcoin.md

```diff
@@ -0,0 +1,172 @@
+# Chapitre 1 : Présentation du Bitcoin
+
+Avant de pouvoir commencer à programmer Bitcoin (et Lightning), vous devez avoir une compréhension de base de ce qu'ils sont et de leur fonctionnement. Le présent chapitre vous donne un aperçu de ces notions. De nombreuses autres définitions apparaîtront dans le document lui-même ; elles ne visent qu'à jeter les bases.
+
+## Objectifs de ce chapitre
+
+Après avoir parcouru ce chapitre, un développeur sera en mesure de faire ceci :
+
+- Évaluer les parties du système Bitcoin
+
+
+- Décider de l'utilité d'une chaîne de blocs
+
+
+Les objectifs de ce cours sont de permettre de :
+
+- Comprendre ce qu'est Bitcoin
+
+- Comprendre ce que sont les transactions dans la chaîne de blocs (Blockchain)
+
+- Comprendre ce qu'est une chaîne de blocs
+
+- Comprendre ce qu'est Lightning
+
+## A propos du Bitcoin
+
+Bitcoin est un système programmatique qui permet le transfert de la monnaie bitcoin. Il est activé par un système décentralisé de nœuds pairs-à-pair (ndt : peer-to-peer), qui comprend des nœuds complets, des portefeuilles et des mineurs. En travaillant ensemble, ils s'assurent que les transactions de bitcoin sont rapides et non répudiables. Grâce à la nature décentralisée du système, ces transactions sont également résistantes à la censure et peuvent offrir d'autres avantages tels que le "pseudonymat" et la non-corrélation si elles sont bien utilisées.
```

Comment:
Bitcoin est un système **programmable** qui permet le transfert de la monnaie bitcoin. Il est **rendu possible** par un système décentralisé de nœuds pairs-à-pair (ndt : peer-to-peer), qui comprend des nœuds complets, des portefeuilles et des mineurs. En travaillant ensemble, ils s'assurent que les transactions de bitcoin sont rapides et non répudiables. Grâce à la nature décentralisée du système, ces transactions sont également résistantes à la censure et peuvent offrir d'autres avantages tels que le "pseudonymat" et la non-corrélation si elles sont bien utilisées.

Reply:
Fait